vizro-core/src/vizro/static/js/models/container.js

```diff
@@ -0,0 +1,28 @@
+function toggle_container(n_clicks, is_open) {
```

Comment:
Please can you add a short comment referencing the `collapse_nav_panel` callback just to explain their similarities/differences?